packages/composables/src/hooks/useCurrency.ts

```diff
@@ -0,0 +1,96 @@
+import Vue from "vue";
+import { reactive, computed, onMounted, Ref } from "@vue/composition-api";
+import {
+  getAvailableCurrencies,
+  setCurrentCurrency,
+  getSessionContext,
+} from "@shopware-pwa/shopware-6-client";
+const sharedCurrency = Vue.observable({
+  currentCurrency: null,
+  availableCurrencies: [],
+} as any);
+
+/**
+ * @alpha
+ */
+export interface UseCurrency {
+  onMountedCallback: () => Promise<void>;
+  switchCurrency: (currencyId: string) => Promise<boolean>;
+  availableCurrencies: Ref<Readonly<any>>;
+  currentCurrency: Ref<Readonly<any>>;
+  currentCurrencySymbol: Ref<Readonly<any>>;
+}
+
+/**
+ * @alpha
+ */
+export const useCurrency = (): UseCurrency => {
+  const localCurrency = reactive(sharedCurrency);
```

Comment:
we'll add here something like 
```ts
const {currency, setCurrency} = useSessionContext()
```

then currency we can export right away, and add only compute for currencySymbol

Reply:
:+1: